### ISSUE TITLE
[release-test] rllib_learning_tests_es_(tf|torch).aws failed

### ISSUE THREAD

**Issue by can-anyscale** [ray-project/ray#35461]
### What happened + What you expected to happen

rllib_learning_tests_es_(tf|torch).aws failed and is blocking ray 2.5 release. Please help review and resolve ASAP.

I kicked off a bisect here: https://buildkite.com/ray-project/release-tests-bisect/builds/159

### Versions / Dependencies

master, 2.5 release

### Reproduction script

https://buildkite.com/ray-project/release-tests-branch/builds/1658#01882a6d-b9be-4a55-baf2-0d6f965e23f3

### Issue Severity

High: It blocks me from completing my task.

**Comment by can-anyscale** [comment]
Bisect blame this https://github.com/ray-project/ray/commit/ea6b926605566f3c6d95f3849771ad048409adde, CC: @ArturNiederfahrenhorst

**Comment by can-anyscale** [comment]
Assign this to you then @ArturNiederfahrenhorst , should we just revert the PR or it is fixing another issue and needs to fix forward?

**Comment by can-anyscale** [comment]
Since this is a release-blocker issue, please **close it only after the cherry pick fix is merged into 2.5 release branch**. 

Please add @ArturNiederfahrenhorst as one of the reviewer of the fix as well for tracking purpose. Thankks!

**Comment by can-anyscale** [comment]
These tests passed on release branch, so removing the release-blocker tag
